docs/howto/writing-an-application-from-scratch/src/main/resources/HelloWorldEndpoint.java

```diff
@@ -0,0 +1,16 @@
+package com.example.rest;
+
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.GET;
+import javax.ws.rs.Produces;
+
+@Path("/hello")
+public class HelloWorldEndpoint {
+
+  @GET
+  @Produces("text/plain")
+  public Response doGet() {
+    return Response.ok("Hello from WildFly Swarm!").build();
```

Comment:
Not sure how far we want to get in this simple example, but you can also just return a string:

```java
@GET
@Produces("text/plain")
public String get() {
    return "Hello from WildFly Swarm!";
}